### ISSUE TITLE
AttributeError raised when misspellings are present (4.0.0)

### ISSUE THREAD

**Issue by reaperhulk** [sphinx-contrib/spelling#5]
As of 4.0.0 (and with sphinx 1.6.4) the following exception is raised when a misspelling occurs:

```
Exception occurred:
  File "/path/to/lib/python2.7/site-packages/sphinxcontrib/spelling/builder.py", line 150, in finish
    self.warning('Found %d misspelled words' % self.misspelling_count)
AttributeError: 'SpellingBuilder' object has no attribute 'warning'
```

This commit (https://github.com/sphinx-contrib/spelling/commit/9f9768e48d51e7d3e0a5feebcf1402447c40fd95) appears to have introduced the issue.

**Comment by markuszoeller** [comment]
Same issue here.

**steps to reproduce**
```
git clone https://github.com/markuszoeller/markuszoeller.github.io.git
cd markuszoeller.github.io
./build_sources.sh
source .venv/bin/activate
./check_spelling.sh 
```

**debug data**
[markuszoeller_spelling_4_issue.txt](https://github.com/sphinx-contrib/spelling/files/1375308/markuszoeller_spelling_4_issue.txt)
[sphinx-err-lamtQF.log](https://github.com/sphinx-contrib/spelling/files/1375309/sphinx-err-lamtQF.log)

**Comment by dhellmann** [comment]
This should be fixed in 4.0.1, please let me know if it is not.

**Comment by markuszoeller** [comment]
v4.0.1 works like a charm, thanks!
